### PR TITLE
Fixes for facebook new design

### PR DESCRIPTION
**Pull request by Myshor**
Forcing dark fixes into light mode.
Fixing rect for groups.

Still to find solution:
dynamicly added elements like expanded groups, new contacts on the list are still darkened.

### CODE REVIEW

**Comment by Myshor** [comment]
Added darkening background pics in new design:
image

**Comment by Myshor** [comment]
Refactor for darkening to have same results as old design.
image

**Comment by Myshor** [comment]
Added fixes for movies slider:
image

**Comment by Myshor** [comment]
Added fix to remove unwanted background in video controls 'caused by darkening images in posts with text on image.

**Comment by Myshor** [comment]
Final fix for darkening images:
image

But to not have dark background in movies controls:
image

**Comment by Myshor** [comment]
Added fix for icons in memories (forced original file instead of inverted blob).
Added fix for white circle in light/dark mode switch.
image

**Comment by Myshor** [comment]
Added fixes for stories and secondary button background:
image

Add highlight for buttons on hover:
image

**Comment by Myshor** [comment]
Fix for menu and main icons highlight on hover.
image
image

**Comment by Myshor** [comment]
Fix for white rects.

Before:
image
After:
image

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -928,6 +928,60 @@ CSS
...
+    --filter-placeholder-icon: invert(59%) sepia(11%) saturate(200%) saturate(135%) hue-rotate(176deg) brightness(96%) contrast(94%) !important;
+    --filter-primary-icon: invert(100%) !important;
+    --filter-secondary-icon: invert(62%) sepia(98%) saturate(12%) hue-rotate(175deg) brightness(90%) contrast(96%) !important;
+    --filter-warning-icon: invert(77%) sepia(29%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(128%) hue-rotate(359deg) brightness(102%) contrast(107%) !important;
+    --filter-blue-link-icon: invert(73%) sepia(29%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(103.25%) hue-rotate(189deg) brightness(101%) contrast(101%) !important;
+    --filter-positive: invert(37%) sepia(61%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(115%) hue-rotate(91deg) brightness(97%) contrast(105%) !important;
+    --filter-negative: invert(25%) sepia(33%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(200%) saturate(110%) hue-rotate(345deg) brightness(132%) contrast(96%) !important;
+}
+g > rect {
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:941]: Maybe there could be a more specific selector?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -928,6 +928,60 @@ CSS
...
+    --darkreader-inline-fill: none !important;
+}
+mask > rect, mask > circle {
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:944]: Maybe there could be a more specific selector?

**Comment by Myshor** [comment]
grayed mode switch and appearing buttons for photo edit while creating post
.q2y6ezfg
image

**Comment by Myshor** [comment]
Fix for non-default colors in groups:
image

**Comment by Myshor** [comment]
Background for profile button added:
image

**Comment by Myshor** [comment]
Fix for inverted fundraisers page in native FB dark mode:
before
image
after
image

**Comment by Myshor** [comment]
OK... native dark checked. Tomorrow I will merge this. 😉

**Pull request merged**